lib/shared_print/deprecation_record.rb

```diff
@@ -77,8 +79,8 @@ def org_commitments
     # If commitments contain deprecated ones, add to @err and remove them.
     # Return only non-deprecated
     def reject_deprecated
-      already_deprecated = org_commitments.select(&:deprecated?)
-      @org_commitments.reject!(&:deprecated?)
+      already_deprecated = org_commitments&.select(&:deprecated?)
+      @org_commitments&.reject!(&:deprecated?)
       if @err.empty? && org_commitments.empty?
```

Comment:
so I guess we don't need `&.` here because if `@err.empty?` is true then `org_commitments` will be an array?

Reply:
No more &.